### DISCUSSION TITLE
Specify faces to look out for

### DISCUSSION THREAD

**Discussion by flolau2k** [nextcloud/recognize#450] [General]
I've been using recognize for a few days now for faces and it works pretty well, in fact so well that it finds way to many faces in all my photos. 
I am wondering would it be possible to specify the faces to look out for so that only those will be recognized and listed?

**Comment by marcelklehr** [comment]
Faces should be sorted by most instances found in the photos app, so the most important faces should be up top. There's also a fix in the works that will improve speed of face loading in the photos app. So, this should become less of an issue.
